src/encoded/tests/test_schemas.py

```diff
@@ -0,0 +1,196 @@
+import pytest
+import re
+
+from pkg_resources import resource_listdir
+from snovault import COLLECTIONS, TYPES
+from snovault.schema_utils import load_schema
+
+
+pytestmark = [pytest.mark.setone, pytest.mark.working, pytest.mark.schema, pytest.mark.indexing]
+
+
+SCHEMA_FILES = [
+    f for f in resource_listdir('encoded', 'schemas')
+    if f.endswith('.json') and 'embeds' not in f
+]
+
+
+def pluralize(name):
```

Comment:
I would like to think this does the same kinds of things as dcicutils.lang_utils.pluralize, or that if it does something that function doesn't that we fix that.